test/mysql2_test.rb

```diff
@@ -34,7 +34,7 @@ def test_semian_can_be_disabled
   end
 
   def test_connection_errors_open_the_circuit
-    @proxy.downstream(:latency, latency: 1200).apply do
+    @proxy.downstream(:latency, latency: 2200).apply do
```

Comment:
Curious why you had to change this?

Reply:
I upped the default timeout to 2s, to show that it changes to 1s during the half open state.